frontend/www/app/app.js

```diff
@@ -9,6 +9,7 @@ angular.module('civis.youpower.donation',[]);
 
 angular.module('civis.youpower', [
   'ionic',
+  'ion-datetime-picker',
```

Comment:
The ion-datatime-picker is not included anywhere and breaks the production! Please, please in future make sure to inlucde new libraries!